docs/source/conf.py

```diff
@@ -33,9 +33,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "Arrow DataFusion"
-copyright = "2022, Apache Software Foundation"
-author = "Arrow DataFusion Authors"
+project = "Apache Arrow DataFusion"
+copyright = "2019-2024, Apache Software Foundation"
```

Comment:
Should the starting year remain 2022 or be 2019?

Reply:
I think 2019 is good, because that was when the project became part of Apache Arrow